.github/workflows/push-bundles.yaml

```
@@ -38,6 +38,7 @@ jobs:
         go-version-file: go.mod
         cache: true
 
+    # TODO: Remove this once all referecnes to quay.io/hacbs-contract are removed
```

(same typo as in ec-cli)
```suggestion
    # TODO: Remove this once all references to quay.io/hacbs-contract are removed
```